### PR TITLE
docs: additional description to download different versions of APISIX

### PR DESCRIPTION
**Pull request by bin-ya**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/how-to-build.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ sudo yum --showduplicates list apisix
 
 # Will install the latest apisix package
 sudo yum install apisix
+
+# Will install the specified version apisix package
+sudo yum install apisix-x.xx.x-0.el7
```
Comment (tzssangglass) [inline docs/en/latest/how-to-build.md:59]: It is better to use a specific version instead of  apisix-x.xx.x-0.el7

**Comment by moonming** [comment]
this PR is the same as https://github.com/apache/apisix/pull/6519

File: docs/en/latest/how-to-build.md [hunk 2]
```diff
@@ -54,6 +54,9 @@ sudo yum --showduplicates list apisix
 
 # Will install the latest apisix package
 sudo yum install apisix
+
+# Will install the specified version 2.10.3 apisix package
```
Comment (spacewander) [inline docs/en/latest/how-to-build.md:58]:
```suggestion
# Will install a specified version (2.10.3 in this example) apisix package
```

**Review by tzssangglass — DISMISSED** [review]

**Review by spacewander — COMMENTED** [review]
Let's update zh version: https://github.com/apache/apisix/blob/master/docs/zh/latest/how-to-build.md

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**